assets/boilerplate/theme.json

```diff
@@ -0,0 +1,303 @@
+{
+    "$schema": "https://schemas.wp.org/trunk/theme.json",
+    "settings": {
+        "appearanceTools": true,
+        "border": {
+            "color": true,
+            "radius": true,
+            "style": true,
+            "width": true
+        },
+        "color": {
+            "custom": true,
+            "customGradient": true,
```

Comment:
Custom color and custom gradients are enabled by default, so these two lines can be removed.
https://developer.wordpress.org/block-editor/reference-guides/theme-json-reference/theme-json-living/#color